src/main/java/com/iexec/sms/secret/app/AppRuntimeSecret.java

```diff
@@ -20,21 +20,24 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
-import org.hibernate.annotations.GenericGenerator;
 
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
 @NoArgsConstructor
 @Entity
-public class ApplicationRuntimeSecret extends Secret {
+public class AppRuntimeSecret extends Secret {
```

Comment:
I don't think it fits into this story. I'd prefer to add it in a later PR so that what's done there is clear.

Reply:
I seems you both refactored and renamed classes/method so it's difficult for Git and us to see what is new

Reply:
Yes, I agree with you. The idea was to put all the renaming into the first commit so that we could exclude it. But we added another renaming commit (https://github.com/iExecBlockchainComputing/iexec-sms/pull/69/commits/f725d938388f8a40807ac64191456a738fa56872) so it became hard to read. I'll try to move all the renaming into a separate PR then rebase the current PR onto this new one. Is this OK for you?